packages_rs/nextclade-web/src/components/Results/ListOfQcIsuues.tsx

```diff
@@ -61,6 +63,7 @@ export function ListOfQcIssues({ qc }: ListOfQcIssuesProps) {
     { name: t('Mutation Clusters'), shortName: 'C', value: snpClusters, message: formatQCSNPClusters(t, snpClusters) }, // prettier-ignore
     { name: t('Frame shifts'), shortName: 'F', value: frameShifts, message: formatQCFrameShifts(t, frameShifts) }, // prettier-ignore
     { name: t('Stop codons'), shortName: 'S', value: stopCodons, message: formatQCStopCodons(t, stopCodons) }, // prettier-ignore
+    { name: t('Coverage'), shortName: 'c', value: coverage, message: formatQCCoverage(t, coverage) }, // prettier-ignore
```

Comment:
The letter inside the circle in the UI is defined here. Uppercase "C" was already used, so I figured lowercase "c" might be fine.